tools/eval/datasets/imagenet.py

```diff
@@ -63,3 +64,1004 @@ def get_result(self):
     def print_result(self):
         print("Top-1 Accuracy: {:.2f}%; Top-5 Accuracy: {:.2f}%".format(self.top1_acc*100, self.top5_acc*100))
 
+
+    LABELS_IMAGENET_1K = '''tench
```

Comment:
We dont need these labels for evaluation.

Reply:
But the `model.infer(img)` will give labels directly not index of labels.

Reply:
I suggest to add `load_label=True` in for example `PPResNet` then set it `False` in evaluation.

Reply:
I will create another PR to modified it.

Reply:
What's the difference between `text_recognition_CRNN_EN_2021sep.onnx` and `text_recognition_CRNN_EN_2023feb.onnx`? They have the same accuracy through evaluation.